### DISCUSSION TITLE
Reverse cards without sentences

### DISCUSSION THREAD

**Discussion by EtienneHauber** [Jitendex/Jitendex#111] [Q&A] [answered]
Hey there, I like to make cards of 'vocab-in-the-wild' - Yomitan is perfect for this and I use Jitendex. However, I noticed that the beautiful glossary it generates includes just a bit too much info for me to easily create "reverse" cards - especially the sentences are spoilers. Is it possible to remove the sentences from the glossary?

**Comment by stephenmk** [comment, marked as answer]
You can add a custom style to your anki cards to hide sentences.

```css
[data-sc-content|="example-sentence"] {
  display: none;
}
```

If you're not sure how to add custom styles, I wrote a quick guide here: https://github.com/stephenmk/Jitendex/issues/66#issuecomment-1913696575

**Reply by EtienneHauber** [reply]
You are amazing, thank you for your quick answer and good work!!
